docs/testplan/pac/Port_Access_Control.md

```diff
@@ -0,0 +1,566 @@
+
+## 1 Test Focus Areas
+
+### 1.1 CLI Testing
+
+	- Verify port authentication can be enabled only on physical interfaces and gets denied on VLAN, Portchannel, PO member ports and sub interfaces.
+	- Verify configured CLI fields are updated properly in respective show commands
+	- Verify PAC configurations/show outputs via REST URI
+
+### 1.2 Functional Testing
+
+	- Verify all data traffic is blocked when PAC is enabled on the port.
+	- Verify 802.1x client authentication in single-host mode and verify only first authenticated user is allowed.
+	- Verify 802.1x client with multi-host mode and verify all users on the port are allowed after first-user gets authenticated.
+	- Verify 802.1x client authentication in multi-auth mode and verify all users with valid credentials gets authenticated.
+	- Verify in multi-auth mode, one of the clients logoff does not impact other authenticated clients.
+	- Verify in multi-host mode, if primary host gets logged-off, other hosts are blocked and verify after authenticating again.
+	- Verify 802.1x client authentication with port-control mode as force-authorized.
+	- Verify 802.1x client authentication with port-control mode as force-unauthorized.
+	- Verify interface level 802.1x pae authenticator disable/enable.
+	- Verify global 802.1x system-auth-control enable/disable.
+	- Verify enabling re-authentication with different re-authenticate timer and disabling authentication periodic shouldn't allow re-authentication.
+	- Verify client authentication with MAB auth-type as EAP-MD5.
+	- Verify client authentication with MAB auth-type as PAP.
+	- Verify client authentication with MAB auth-type as CHAP.
+	- Verify authentication order with user-configured priorities for different authentication methods.
+	- Verify non-default max-users per port and check remaining clients are denied.
+	- Verify that when host mode changes, the authenticated clients gets removed and traffic is blocked.
+	- Verify that when authentication order is set to 802.1x, then only 802.1x client allowed to authenticate.
+	- Verify that when authentication order is set to MAB, then 802.1x client authentication is not successful.
+	- Verify MAB client authentication with port-control mode toggle between force authorized/unauthorized and auto.
+	- Verify that 802.1x and MAB client is not authenticated if it's RADIUS assigned VLAN is not available statically on the authenticator switch.
+	- Verify 802.1x and MAB client is not authenticated if RADIUS does not assign a VLAN and the port is configured with tagged VLAN.
+	- Verify a port with Multi-auth mode can have authenticated clients in different radius assigned VLANs.
+	- Verify that 802.1x and MAB client is not authenticated if RADIUS does not assign a VLAN and the port's configured untagged VLAN (Access VLAN) is not available
+
```

Comment:
I assume these testcases use FreeRADIUS server, pls put a note

Reply:
Addressed the comments